docs/sources/developers/plugins/backend/plugin-protocol.md

```diff
@@ -20,7 +20,7 @@ Because Grafana maintains the plugin protocol, the plugin protocol attempts to f
 
 ## Writing plugins without Go
 
-In case you want to write a backend plugin in another language than Go it’s possible as long as the language supports [gRPC](https://grpc.io/). However, writing a plugin in Go is recommended and has several advantages that should be carefully taken into account before proceeding:
+In case you want to write a backend plugin in another language than Go, it’s possible as long as the language supports [gRPC](https://grpc.io/). However, writing a plugin in Go is recommended and has several advantages that should be carefully taken into account before proceeding:
```

Comment:
```suggestion
If you want to write a backend plugin in another language than Go, then it’s possible as long as the language supports [gRPC](https://grpc.io/). However, writing a plugin in Go is recommended and has several advantages that should be carefully taken into account before proceeding:
```

I think we can improve it even more! If/then is easier for browser to translate for better accessibility. WDYT?